src/tasks/prettier-format.ts

```diff
@@ -1,15 +1,52 @@
 import { execa } from "execa";
+import path from "path";
+import { Options } from "../types";
+import { SOLIDITY_FRAMEWORKS } from "../utils/consts";
 
-// TODO: Instead of using execa, use prettier package from cli to format targetDir
-export async function prettierFormat(targetDir: string) {
+async function runPrettier(targetPath: string, prettierConfigPath: string, prettierPlugins: string[]) {
+  const result = await execa("yarn", [
+    "prettier",
+    "--write",
+    targetPath,
```

Comment:
changing `targetPath` to `string[]` and passing `...targetPath` here fixes the error.

we can also change `targetPath` to `string | string[]` but it will allow to path strings with spaces which can cause errors so I think just array is better